src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -445,6 +494,68 @@
     crafted: true
 """
 
+helps['network application-gateway settings'] = """
+type: group
+short-summary: Manage settings of an application gateway.
+"""
+
+helps['network application-gateway settings create'] = """
+type: command
+short-summary: Create settings.
+examples:
+  - name: Create settings.
+    text: |
+        az network application-gateway settings create -g MyResourceGroup --gateway-name MyAppGateway \\
+            -n MySettings --port 80 --protocol Http --timeout 30
+  - name: Create settings. (autogenerated)
+    text: |
+        az network application-gateway settings create --gateway-name MyAppGateway --host-name MyHost --name MySettings \
+        --port 80 --probe MyNewProbe --protocol Tcp --resource-group MyResourceGroup
+    crafted: true
+"""
+
+helps['network application-gateway settings delete'] = """
+type: command
+short-summary: Delete settings.
+examples:
+  - name: Delete settings.
+    text: az network application-gateway settings delete -g MyResourceGroup --gateway-name MyAppGateway -n MyHttpSettings
+"""
+
+helps['network application-gateway settings list'] = """
+type: command
+short-summary: List settings.
+examples:
+  - name: List settings.
+    text: az network application-gateway settings list -g MyResourceGroup --gateway-name MyAppGateway
+"""
+
+helps['network application-gateway settings show'] = """
+type: command
+short-summary: Get the details of a gateway's settings.
+examples:
+  - name: Get the details of a gateway's settings.
+    text: az network application-gateway settings show -g MyResourceGroup --gateway-name MyAppGateway -n MySettings
+"""
+
+helps['network application-gateway settings update'] = """
+type: command
+short-summary: Update settings.
+examples:
+  - name: Update settings to use a new probe.
+    text: |
+        az network application-gateway settings update -g MyResourceGroup --gateway-name MyAppGateway \\
+            -n MySettings --probe MyNewProbe
+  - name: Update settings. (autogenerated)
+    text: |
+        az network application-gateway settings update --gateway-name MyAppGateway --name MySettings --probe MyNewProbe --resource-group MyResourceGroup
+    crafted: true
+  - name: Update settings. (autogenerated)
+    text: |
+        az network application-gateway settings update --gateway-name MyAppGateway --host-name-from-backend-pool true --name MySettings --port 80 --probe MyNewProbe --resource-group MyResourceGroup
+    crafted: true
```

Comment:
Why these two examples have the same name?

Reply:
I will revise it.